changelog/shared-bitmapped-block.dd

```diff
@@ -0,0 +1,8 @@
+Added the SharedBitmappedBlock, as the threadsafe version of the regular BitmappedBlock.
+
+The limitation of the new implementation is that the shared version can only allocate 
+at most one block per allocation. This however allows for greater performance, because
+the synchronization is done at a single block level, whereas for multiple blocks, the whole
+bit array should have been synchronized by means of wide locking.
+
+
```

Comment:
- Empty lines at the end
- Link the new symbol once in the first sentence: `$(REF SharedBitmappedBlock, std,experimental,allocator,building_blocks,bitmapped_block)`